msg/precision_landing_status.msg

```diff
@@ -0,0 +1,10 @@
+# ORBIT_YAW_BEHAVIOUR
```

Comment:
ORBIT_YAW_BEHAVIOUR

Reply:
Hi @potaito , I's love to see this PR make it in.  Do you want to commit above suggestion and rebase on upstream?

Reply:
Sure can do :+1: 
I was expecting more comments. This can't be the only thing that's not optimal :joy: 

As far as I know, PrecisionLanding is not used a lot by the community, and the folks who use it probably have their own implementation of it. So maybe there is someone who would be unhappy with this PR, but since the PrecisionLanding is then a flight task, customizing and changing it would be super simple.

Reply:
Done, there were only minor conflicts when rebasing.